raw_socket_listener/tcp_message.go

```diff
@@ -92,9 +92,14 @@ func (t *TCPMessage) IsMultipart() bool {
 	}
 
 	payload := t.packets[0].Data
-	m := payload[:4]
```

Comment:
But this way you would miss this packet and possibly break the TCP flow. Am I correct?